core/client/bower.json

```diff
@@ -23,7 +23,7 @@
     "jqueryui-touch-punch": "furf/jquery-ui-touch-punch",
     "jquery.simulate.drag-sortable": "0.1.0",
     "keymaster": "1.6.3",
-    "loader.js": "3.2.1",
+    "loader.js": "3.3.0",
```

Comment:
Is there are particular reason we're not specifying this the same way as ember-cli? `"loader.js": "ember-cli/loader.js#3.4.0",` - there must be a reason why they're explicit about which `loader.js` to pull in.